### DISCUSSION TITLE
segregated crossing

### DISCUSSION THREAD

**Discussion by ivanbranco** [streetcomplete/StreetComplete#4814] [General]
I have some doubts when SC asks me if this is segregated or not, since there's no physical separation: 
![crossing](https://user-images.githubusercontent.com/94002623/218881611-0d2d7c24-44c6-4338-8656-18350f7d9ff9.jpg)

In the Key:segregated page there's no mention about physical separation but the only image is physically separated so I'm not sure.

**Comment by matkoniecz** [comment]
> I have some doubts when SC asks me if this is segregated or not, since there's no physical separation:

It is also segregated.

> In the Key:segregated page there's no mention about physical separation but the only image is physically separated so I'm not sure.

I added https://commons.wikimedia.org/wiki/File:Krak%C3%B3w_-_Bulwar_Inflancki_-_lower_level.jpg as an example image there that show painted line only to https://wiki.openstreetmap.org/wiki/Key:segregated#Examples
